### ISSUE TITLE
isn't working on me(anigame sniper)

### ISSUE THREAD

**Issue by Maru-Yasa** [Sebastian09-09/Anigame-Sniper-Gen-2#4]
So i put the code to heroku and run frontend.py and also beckend.py, then the console looks working, they return like i run it locally,
but in my case isn't claim any anigame's drop, then i try to fix it by myself but i cant find the code that functionally claimming in both of the frontend and backend file, can you explainning your code? or just figure it out why isn't working on me?
![herokulogs-frontend](https://user-images.githubusercontent.com/41475719/132605024-4ad7d6f2-a568-4939-ac9f-b3908a91cde9.png)
![herokulogs-backend](https://user-images.githubusercontent.com/41475719/132605029-9df535a6-c678-4670-beb6-7a971971121d.png)

**Comment by Sebastian09-09** [comment]
did you turn on the sniper ?
by typing $add / {your prefix}add ?

**Comment by Maru-Yasa** [comment]
yes i did (turn on the sniper), you can see "sniper is on in 5 channels!" it mean that the sniper is alredy on right?

and yes, two files hosted in the same herokuapp, i edit the Procfile so that it can run 2 dynos at the sametime
![nano](https://user-images.githubusercontent.com/41475719/132779623-53f3333f-ef32-421d-aa03-a7ae0f68cacd.png)
![free dtynos](https://user-images.githubusercontent.com/41475719/132779507-f14902a9-c2d9-4de0-ab20-f0d41980822f.png)

**Comment by Sebastian09-09** [comment]
everything runs fine for me when i run it locally and i dont know how to setup heroku so that it runs 2 files in the same app , if you can tell me how to or maybe refer me a video about running 2 files in the same herokuapp  , i can debug the sniper
